src/elements/content-uploader/ContentUploader.js

```diff
@@ -1203,17 +1205,19 @@ class ContentUploader extends Component<Props, State> {
      */
     render() {
         const {
+            className,
+            fileLimit,
+            isDraggingItemsToUploadsManager = false,
+            isFolderUploadEnabled,
+            isResumableUploadsEnabled,
+            isTouch,
             language,
+            maxFileSize,
+            measureRef,
             messages,
             onClose,
-            className,
-            measureRef,
-            isTouch,
-            fileLimit,
+            onFreemiumUpsellCTAClick,
```

Comment:
ContentUploader can used by platform developers in their own applications, outside of Box's webapp. We should try to keep the component API simple and general, when possible. Do we plan to add more experiences like this one? If so, should we wrap them in an object to avoid further prop bloat?

Reply:
These are the final current planned ContentUploader changes for this project. No other prop needed. I suspect that once the experiment concludes, I will make a PR to clean up these changes (2 props)